pennylane_lightning_gpu/src/simulator/cuGateCache.hpp

```diff
@@ -53,48 +52,44 @@ template <class fp_t> class GateCache {
      *
      */
     void defaultPopulateCache() {
+        using namespace Pennylane::CUDA::Util;
+
         host_gates_[std::make_pair(std::string{"Identity"}, 0.0)] =
-            std::vector<CFP_t>{cuUtil::ONE<CFP_t>(), cuUtil::ZERO<CFP_t>(),
-                               cuUtil::ZERO<CFP_t>(), cuUtil::ONE<CFP_t>()};
+            std::vector<CFP_t>{ONE<CFP_t>(), ZERO<CFP_t>(), ZERO<CFP_t>(),
+                               ONE<CFP_t>()};
```

Comment:
I assume we do not need to worry about clashes with the Lightning CPU implementation?

Reply:
Shouldn't be the problem. It's worth double-checking; thanks for bringing it up.